### DISCUSSION TITLE
Unable to load saved scenes

### DISCUSSION THREAD

**Discussion by HarshithaBR** [empriselab/RCareWorld#182] [Help - Programming]
Dear Maintainers.

I am able to save a scene as json using the saveScene() method. However; I am unable to load the saved scene, as I want to build a script on top of the saved scene. 

I am using the below code to load the scene:
``````
env.step()
env.LoadSceneAsync("dressone.json")
print("loaded the scene")
env.step(20)
``````

Can you please share a work around for this, or how I can get this functionality to work?

Thanks and best regards
Harshitha B R

**Comment by YoruCathy** [comment]
Hi @HarshithaBR , try this one https://github.com/empriselab/RCareWorld/blob/1cb52f5333ebc1c191f442346beb865198dd6ac7/pyrcareworld/pyrcareworld/demo/examples/example_light.py#L30

You can directly initialize the scene with the json file
